arviz/stats/diagnostics.py

```diff
@@ -996,3 +996,136 @@ def _multichain_statistics(ary, focus="mean"):
         ess_tail_value,
         rhat_value,
     )
+
+
+def psens(data, *, component, var_names=None, delta=0.01, dask_kwargs=None):
+    """Compute power-scaling sensitivity diagnostic.
+
+    Power-scales the prior or likelihood and calculates how much the posterior is affected.
+
+    Parameters
+    ----------
+    data : obj
+        Any object that can be converted to an :class:`arviz.InferenceData` object.
+        Refer to documentation of :func:`arviz.convert_to_dataset` for details.
+        For ndarray: shape = (chain, draw).
+        For n-dimensional ndarray transform first to dataset with ``az.convert_to_dataset``.
+    var_names : list
+        Names of variables to include in the rhat report
+    component : str
+        Select component to power-scale. Valid components are are:
+        - "prior"
+        - "likelihood"
+    dask_kwargs : dict, optional
+        Dask related kwargs passed to :func:`~arviz.wrap_xarray_ufunc`.
+
+    Returns
+    -------
+    xarray.Dataset
+      Returns dataset of power-scaling sensitivity diagnostic values.
+
+    Notes
+    -----
+    The diagnostic is computed by power-scaling the specified component (prior or likelihood)
+    and determining the degree to which the posterior changes. It uses Pareto-smoothed
```

Comment:
```suggestion
    The diagnostic is computed by power-scaling the specified component (prior or likelihood)
    and determining the degree to which the posterior changes as described in [1]_. It uses Pareto-smoothed
```

moreover, the notes should explain that this uses either the `log_prior` or `log_likelihood` variables in `sample_stats` and that therefore they should contain the log probabilities corresponding to the level we want to diagnose, leading to the `log_prior + log_likelihood` not necessarly being `lp`.